readme.md

```diff
@@ -151,6 +151,15 @@ new Clipboard('.btn', {
 });
 ```
 
+For use in bootstrap modals you'll want to set the modal element as the `container` value.
+See [Issue #155 (comment)](https://github.com/zenorocha/clipboard.js/issues/155#issuecomment-273124130)
+
+```js
+new Clipboard('.btn', {
+    container: document.getElementById('#modal')
+});
+```
```

Comment:
This issue isn't exclusive to bootstrap — you get the same issue with `react-modal2` or any other library that changes the focus.

Might be worth clarifying this?